CONTRIBUTING.md

```diff
@@ -1 +1,73 @@
+    - ❌ Issues are not appropriate if they meet any of the following criteria:
+        - already assigned to someone
+        - labeled with a `restricted: ...` label
+        - labeled with any `status: ...` label other than `status: ready`
+    - ⚠️ Some issues fall into a middle ground, not being labeled "help wanted" or "restricted". These tickets are more challenging and are only appropriate for community contributors who are familiar with our codebase.
+
+    If you want to work on something for which there is no GitHub issue open yet, create an issue and propose your change there. A Mathesar [team member](https://wiki.mathesar.org/en/team) will evaluate your issue and decide whether we'll accept a pull request for the issue.
+
+1. ***(Optionally)* Claim the issue.**
+
+    If the Mathesar team has already merged one of your PRs, then you may wish to ear-mark the issue for yourself so that other do not work on it concurrently.
+    
+    However, **if you are brand new to Mathesar, we recommend skipping this step** and making your changes *without* claiming the issue. (A core team member will assign the issue to you *after* you open a PR.) This recommendation is designed to reduce the overhead the core team has experienced from a high volume of contributors failing to follow through with their intent to submit PRs.
+
+    If you decide to claim the issue:
+
+    1. Comment on the ticket, saying *"I'd like to work on this"* or similar. If you're relatively new to the community, mentioning your PR(s) that we have already merged would be helpful.
+    1. A core team member will assign you to the ticket.
+    1. At this point **you have one week to follow up.** If we don't hear from you by then, we will unassign you from the ticket so that others may claim it. If you need more time, you can ask for an extension, explaining the progress you've made and the challenges you've encountered. If you have not begun work at all, then we will need to unassign you.
+
+    Please do not claim more than 2 issues concurrently before submitting PRs.
+
+1. **Begin making your changes.**
+
+    - Refer to our **[Developer Guide](./DEVELOPER_GUIDE.md)** for questions about the code.
+    - Make sure to follow our [front end code standards](./mathesar_ui/STANDARDS.md) and [API standards](./mathesar/api/STANDARDS.md) where applicable.
+    - If you are not familiar with GitHub or pull requests, please follow [GitHub's "Hello World" guide](https://guides.github.com/activities/hello-world/) first. Make sure to commit your changes on a new git branch named after the ticket you claimed. Base that new branch on our `develop` branch.
+    - Commit early, commit often. Write good commit messages. Try to keep pull requests small if possible, since it makes review easier.
```

Comment:
I like that idea. However, the goal of this PR is to reorganize content while making only a modicum of meaningful edits to the actual content itself. Since this "Commit early" content is [unchanged](https://github.com/centerofci/mathesar-wiki/blob/41309a88b87a45f00f7d57bd5cffe59113876513/community/contributing.md?plain=1#L52), I'd rather not touch it in this PR. If you have a good example or tutorial you'd like to link to, can you make another PR on top of this PR which adds that link? That way, we can have a more targeted discussion about that change if needed, and we won't need to hold up this reorganizing work for its sake.

Reply:
There's an example linked from our default PR template.

Reply:
Thanks @kgodey. I opened #2861 linking to that article.